src/test/java/com/hashedin/redmask/integration/redshift/RedMaskITTest.java

```diff
@@ -97,9 +96,10 @@ public static void setup() throws SQLException, IOException {
   }
 
   @Before
-  public void createConnections() throws SQLException, IOException {
+  public void createConnections() throws SQLException, IOException, ClassNotFoundException {
     log.info("Creating connection object");
     // Create a connection object using super user.
+    Class.forName("com.amazon.redshift.jdbc.Driver");
```

Comment:
This `"com.amazon.redshift.jdbc.Driver"` should a final String class variable

Reply:
refactored